docs-src/concepts/what-is-a-run-id.md

```diff
@@ -10,24 +10,37 @@ tags:
 
 A Run Id is a globally unique, platform-level identifier for a [Workflow Execution](/workflows#workflow-execution).
 
+The current Run Id is mutable and can change during a [Workflow Retry](retry-policies). You shouldn't rely on storing the current Run Id, or using it for any logical choices, because a Workflow Retry changes the Run Id and can lead to non-determinism issues.
```

Comment:
```suggestion
The current Run Id is mutable and can change during a [Workflow Retry](/concepts/what-is-a-retry-policy). You shouldn't rely on storing the current Run Id, or using it for any logical choices, because a Workflow Retry changes the Run Id and can lead to non-determinism issues.
```